candidates/views/api.py

```diff
@@ -433,3 +433,8 @@ class ComplexPopoloFieldViewSet(viewsets.ModelViewSet):
     queryset = extra_models.ComplexPopoloField.objects.order_by('id')
     serializer_class = serializers.ComplexPopoloFieldSerializer
     pagination_class = ResultsSetPagination
+
+class PersonRedirectViewSet(viewsets.ModelViewSet):
+    queryset = extra_models.PersonRedirect.objects.order_by('id')
+    serializer_class = serializers.PersonRedirectSerializer
+    pagination_class = ResultsSetPagination
```

Comment:
So, I assume that `old_person_id` will always be unique here? Is that true? If so, can we make the `lookup_field` be that, so that the detail route will be more helpful? I ask because this is how I'd expect to use the API (if a candidate is missing, lookup `/api/person_redirects/[old_id]`, but because the namespace is the same (positive integers) there will be cases when `PersonRedirect.pk` will be the same as a `old_person_id`. This means that people might get redirected to the wrong `new_person_id`.

If `old_person_id` isn't unique, then the detail view should return a list I expect, as the above usage will still happen, and could cause confusion otherwise.